Makefile

```diff
@@ -25,7 +25,7 @@ DST := /gopath/src/github.com/gravitational/rigging
 # This directory
 SRC := $(dir $(CURDIR)/$(word $(words $(MAKEFILE_LIST)),$(MAKEFILE_LIST)))
 
-VERSION ?= 0.0.4
+VERSION ?= 0.0.5
```

Comment:
let's use the usual approach we have everywhere else now... like we have in gravity, shall we?

```
VERSION := $(shell git describe --tags --abbrev=8)
```